felix/bpf-gpl/fib_co_re.h

```diff
@@ -235,7 +235,16 @@ static CALI_BPF_INLINE int forward_or_drop(struct cali_tc_ctx *ctx)
 			}
 			if (!cali_rt_is_tunneled(dest_rt)) {
 				CALI_DEBUG("Not a tunnel route for " IP_FMT " at tunnel device", &ctx->state->ip_dst);
+#if CALI_F_L3_DEV
+				// In the case of wireguard, we support host->host encryption.
+				// Remote host route will not be tunneled and we should not be
+				// denying such packets. Hence let them pass to IP stack.
```

Comment:
This is towards HEP `if (CALI_F_TUNNEL && CALI_F_TO_HEP) {` so the packet won't go to IP stack, it is just processed by the device.